sdk/tests/conformance/buffers/buffer-data-and-buffer-sub-data.html

```diff
@@ -76,59 +73,80 @@
     wtu.glErrorShouldBe(gl, gl.INVALID_VALUE,
         "calling bufferData when buffer size is negative should generate INVALID_VALUE");
 
-    gl.bufferData(gl.ARRAY_BUFFER, 0, gl.STATIC_DRAW);
```

Comment:
The removal of this block of code and bufferDataTestWithArrayBuffer() looks like a bad merge. Could you please double-check?